server/lib/tuist/environment.ex

```diff
@@ -39,6 +39,14 @@ defmodule Tuist.Environment do
     @env == :dev
   end
 
+  def staging? do
+    @env == :stag
+  end
+
+  def canary? do
+    @env == :can
+  end
```

Comment:
I'd stay consistent with the abbreviation that we are already using:

```suggestion
  def stag? do
    @env == :stag
  end

  def can? do
    @env == :can
  end
```